docs/src/1_Mechanics/01_PlayerGuide_Full.md

```diff
@@ -545,35 +548,33 @@ one level of *Fatigue*. Reshuffle your deck and draw all Checks with the
 Lower Hand. Each time you need to reshuffle your deck in this way adds
 another level of Fatigue. If you were to suffer a fourth level of Fatigue,
 you are Knocked Out instead. This signifies your character running out of
-Stamina during a long day. 
+Stamina during a long day.
 
-### Discard Mechanics
+### Discard Powers
 
-Each Dealer may have an additional Burn Mechanic. Dealers may discard cards as
+Each Dealer may have one Discard Power. Dealers may discard cards as
 a Minor Action for additional benefits in combat starting at Level 2. For
 every 2 cards discarded, up to a limit of 4, an Upper/Lower Hand +1 is
 granted described below. These effects do not stack; only one may be applied
-to a character at a time.
+to a character at a time. Cards are discarded without being turned over, and cannot
+yeild Fate Cards.
 
-Below is a list of possible Discard Mechanics, determined by Role. Work with your GM if you
-would like to use a different Burn Mechanic instead.  
+Below is a list of possible Discard Powers, determined by Role. Work with your GM if you
+would like to use a different Discard Power instead.  
 <!-- Specifying in combat until we can revsit non-combat uses of agro and thwart -->
 
-- Clubs ♣️, Aggro: Defense Dealers have a strong foundation and can take the big hits.
-  They can choose to *bury* N cards to force all creatures within 2 spaces to make
-  their next Check to attack targets besides the Defense Dealer at Lower Hand (N/2).
+- Aggro: Discard N cards to force all creatures within 2 spaces to make
+  their next Check to attack targets besides the triggering Dealer at Lower Hand (N/2).
 
-- Diamonds ♦️, Intensify: Casters Dealers have explosive damage but they run out of gas
-  really fast. They can choose to *burn* N cards to make their next Check at Upper
-  Hand (N/2).
+- Help: Discard N cards to grant an ally within 6 spaces Upper Hand (N/2) on all Checks
+  for their next turn.
 
-- Hearts ♥️, Help: Life Dealers are fluid and flexible to adapt the situation. If they
-  are within 6 spaces of an ally, they may *toss* 2 cards to grant their ally Upper
-  Hand(N/2) on their next draw.
+- Thwart: Discard N cards and cause an enemy within 6 spaces to
+  make all Checks at Lower Hand (N/2) until the start of the triggering Dealer's next turn.
 
-- Spades ♠️, Thwart: Offense Dealers deploy all sorts of mechanisms for drawing the eye
-  of enemies away from their allies. They can *scrap* N cards and cause an enemy to
-  make Checks at Lower Hand (N/2) until the start of the Offense Dealer's next turn.
+- Intensify: Discard no cards. Your next attack is treated as an Epic Event. Discard as
```

Comment:
I really like this change for Intensify! Would it be the case then that a player would burn through 10+ cards to succeed or do they select the number of cards they want to attempt, saying "I choose 6 cards" and if they miss all 6 it's over? Or they can draw until they Major Success, even if it takes 20+ cards?

Reply:
I'll add "You may choose to end this Epic Event at any time as a failure."

My intent was to let them keep spamming until they got a the +1, even if it meant 5, 10, or more cards

Reply:
I would say 'the first draw is made with whatever upper/lower hand required by the check, the remainder are treated as standard draws' - so that we're still pulling the Nth additional card but it's not some compounded disadvantage